programs/openbook_v2/tests/utils.rs

```diff
@@ -0,0 +1,60 @@
+use std::ffi::OsStr;
+use solana_program::instruction::Instruction;
+use solana_program::pubkey::Pubkey;
+use solana_program_test::BanksClient;
+use solana_sdk::{
+    signature::Keypair, signer::Signer,
+};
+use solana_sdk::transaction::Transaction;
+
+// trying to find direction to protocol-v2
+pub fn get_paths() -> String {
+    let mut path_buf = std::env::current_dir().unwrap();
+    loop {
+        let option_path = path_buf.iter().last();
+        match option_path {
+            None => {
+                panic!("not found directory protocol-v2 in current working directory");
+            }
+            Some(path) => {
+                if path.to_str().unwrap().contains("protocol-v2") {
+                    return path_buf.to_str().unwrap().to_string();
+                } else {
+                    path_buf = path_buf.parent().unwrap().to_path_buf();
+                }
+            }
+        }
+    }
+}
+
+pub async fn create_account_for_type<T>(banks_client: &mut BanksClient, keypair: &Keypair) -> Pubkey {
```

Comment:
why not just `ProgramTest::add_account_with_base64_data`?

this data should all be static so you can just save the data as a str and load all of these predefined accounts on init

Reply:
this is kinda a nit tho, might not even be worth the hassle

Reply:
than i think this is solved ...